.travis.yml

```diff
@@ -1,7 +1,12 @@
 language: node_js
-cache: yarn
+cache:
+  - npm
+  - yarn
```

Comment:
Why are there two package managers used?

Reply:
Good point, that'll teach me to copy and paste! Now only `yarn` is used.